package.json

```
@@ -119,10 +118,11 @@
     "karma-coverage-istanbul-reporter": "2.0.5",
     "karma-jasmine": "2.0.1",
     "karma-jasmine-html-reporter": "1.4.2",
-    "node-sass": "4.11.0",
+    "ng-packagr": "5.3.0",
     "postcss-loader": "3.0.0",
     "raw-loader": "1.0.0",
     "rxjs": "6.4.0",
+    "sass": "^1.22.12",
```

Let's lock this version too, to avoid accidental breakages.
```suggestion
    "sass": "1.22.12",
```

---

Ah! Should have payed more attention to how it was.
Nice one.